src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -86,363 +88,500 @@ const Hole::ThreadDescription Hole::threadDescription[][171] =
 {
     /* None */
     {
-        { "---", 	6.0, 0.0, 0.0 },
+        { "---",    6.0,    0.0 },
     },
 
     /* ISO metric regular */
-    /* ISO metric threaded core hole diameters according to ISO 2306 */
-    // {name, thread diameter, thread pitch, core hole diameter}
+    // {name, thread diameter, thread pitch}

```

Comment:
@chennes the core hole diameter is removed because it seems its only use is removed (see line 1098). @alfrix can you confirm?

Reply:
core-hole/minor-diameter is a byproduct of the thread profile and major diameter,
the table values were used only in one place and it was removed.

1098? that refers to `HoleCutDiameter` that is in fact has been mislabeled it should be `HeadCutDiameter` that is for the counterbore/countersink